docs/config.md

```diff
@@ -131,98 +131,124 @@ does (e.g, `"1d2h4m40.01s"`).
 
 ## Table of Contents
 
-- [`Addresses`](#addresses)
+- [The go-ipfs config file](#the-go-ipfs-config-file)
+  - [Profiles](#profiles)
+  - [Types](#types)
+    - [`flag`](#flag)
+    - [`priority`](#priority)
+    - [`strings`](#strings)
+    - [`duration`](#duration)
+  - [Table of Contents](#table-of-contents)
+  - [`Addresses`](#addresses)
```

Comment:
It looks like you regenerated the whole ToC here. It's more verbose this way, but probably fine. Also, perhaps this ToC come before everything else.

@lidel WDYT?

Reply:
Regenerated ToC is fine, but if we make such change, it is good time to move it to the very top :+1:

Reply:
Moved in https://github.com/ipfs/go-ipfs/pull/8268/commits/247a916155bc5b0529e0cfd29a76b528147dbf46